source/_integrations/binary_sensor.rest.markdown

```diff
@@ -95,6 +95,14 @@ value_template:
     to extract the value.
   required: false
   type: template
+json_attributes:
+  description: "A list of keys to extract values from a JSON dictionary result and then set as sensor attributes. If the endpoint returns XML with the `text/xml`, `application/xml` or `application/xhtml+xml` content type, it will automatically be converted to JSON according to this [specification](https://www.xml.com/pub/a/2006/05/31/converting-between-xml-and-json.html)"
+  required: false
+  type: [string, list]
+json_attributes_path:
+  description: A [JSONPath](https://goessner.net/articles/JsonPath/) that references the location of the `json_attributes` in the JSON content.
+  required: false
+  type: string
```

Comment:
We no longer allow integrations to add or change a platform YAML configuration.

More information on this can be found in Architecture Decision Record:

- ADR-0007: <https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md#decision>

Reply:
Understood. The "problem" here is, that this is actually not meant to change the platform configuration, but allow these keys for `binary_sensor`s within the [`rest` integration](https://www.home-assistant.io/integrations/rest/) -- this would still be accepted, as of ADR-007, right?

As it seems, the only place where the keys of `binary_sensor` and `sensor` are documented is indeed the respective "platform" sites ([RESTful Binary Sensor](https://www.home-assistant.io/integrations/sensor.rest/) and [RESTful ensor](https://www.home-assistant.io/integrations/sensor.rest/))

Reply:
Or should these (the mapping for `binary_sensor` and `sensor`) be included as done in the [Template integration](https://www.home-assistant.io/integrations/template/)? This would completely remove the update from `RESTful Binary Sensor` and make this whole thing eventually conform ADR-007?